.github/workflows/core-checks.yml

```diff
@@ -63,3 +63,23 @@ jobs:
       - name: Posix Network Build Checker
         run: |
             bash workspace/.github/scripts/posix_network_build_checker.sh workspace
+  arm-gcc-build-checker:
+    name: FreeRTOS ARM GCC Build Check
+    runs-on: ubuntu-latest
+    env:
+      ARM_GCC_TOOLCHAIN_URL: "https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2020q2/gcc-arm-none-eabi-9-2020-q2-update-x86_64-linux.tar.bz2?revision=05382cca-1721-44e1-ae19-1e7c3dc96118&la=en&hash=D7C9D18FCA2DD9F894FD9F3C3DC9228498FA281A"
```

Comment:
Also available in the ubuntu repositories... which are mirrored for Github Actions.